SublimeLinter.sublime-settings

```diff
@@ -115,31 +115,34 @@
     "statusbar.show_active_linters": true,
 
     // Global styles for all linters.
-    // - mark_style options:
-    //   "none"
-    //   "fill", "outline",
-    //   "solid_underline", "squiggly_underline", "stippled_underline"
+    // Styles can also be specified per linter.
+    // - mark_style:
+    //   - "none"
+    //   - "fill", "outline",
+    //   - "solid_underline", "squiggly_underline", "stippled_underline"
     //   The underline styles are replaced with outlines when there is
     //   whitespace in the problem region, because underlines aren't drawn
     //   on whitespace (ST issue #137).
-    // - priority determines which mark is shown in the gutter;
-    //   error marks will still always override warning marks.
-    // - scope is used to determine the color, e.g. region.<colorish>,
-    //   where <colorish> is one of
+    // - icon:
+    //   - "circle", "dot" or "bookmark"
+    //   - "none" to remove the icon
+    //   - A path to an icon file like
+    //     "Packages/SublimeLinter/gutter-themes/Blueberry Cross/error.png"
+    //   - One provided by a gutter theme (e.g. "warning" or "error").
+    //     In theme Default: warning, error, cog, x,
+    //     and diamond, heart, pointer, square, star, triangle, which all
+    //     also have an -outline variant.
+    // - scope:
+    //   Is used to determine the color. E.g. region.<colorish>, with one of
     //   redish, orangish, yellowish, greenish, bluish, purplish, pinkish.
-    // - types is an array which can contain "warning" and/or "error".
-    // - codes is an array which can contain error codes provided by a linter.
-    // - icon can be:
-    //   "circle", "dot" or "bookmark"
-    //   or "none" to remove the icon
-    //   or a path to an icon file like
-    //      "Packages/SublimeLinter/gutter-themes/Blueberry Cross/error.png"
-    //   or one provided by a gutter theme (e.g. "warning" or "error")
-    //      in theme Default: "warning", "error", "cog", "x",
-    //      and "diamond", "heart", "pointer", "square", "star", "triangle" all
-    //      also have an -outline variant.
-    // Note: At least one of these targets is required: types or codes.
-    // Styles can also be specified per linter.
+    // - priority:
+    //   Determines which mark is shown in the gutter;
+    //   error marks will still always override warning marks.
```

Comment:
This is incorrect. For 'problems' with the same priority, error will win over warning. You can promote a warning by giving it a higher priority.

Reply:
Good catch, that's indeed no longer true.